lib/manager.rb

```diff
@@ -0,0 +1,114 @@
+require "pry"
+require_relative "room"
+
+module Hotel
+  class Manager
+    attr_accessor :rooms_reservations_hash
+
+    def initialize
+      @rooms_reservations_hash = Room.make_rooms
```

Comment:
I like having this class method in Room.  However since you never make an instance of the Room class, maybe instead make Room a module, or have `make_rooms` as a private factor method in `Manager`.